addons/rose/addon/modifiers/code-mirror.js

```diff
@@ -42,14 +42,16 @@ const _PRESET_DEFAULTS = {
  */
 export default class CodeMirrorModifier extends Modifier {
   _editor;
```

Comment:
Are all the instance variables and methods that start with an underscore, basically custom properties not part of outside apis/libraries?

Reply:
Yes and No.

The `public` / `private` vars in js is a huge topic lol. This code was already there I just refactored so I am talking as I understood the code, but I could be wrong.

Long story short: the `_editor` yes, I understand this var as private (not outside of the API). The parameter `_` within `modifier()` methods it just express that we are receiving a parameter but we are not going to use it, but we have to declare it because we are receiving other parameters in the same function.

i.e.:

```javascript
testMethod(foo, bar, func){
  // Within the testMethod we will use the 3 vars foo, bar, func
}

testMethod(foo, _, func){
  // Within the testMethod we will use JUST 2 vars, foo and func
}
```

Not sure this answers your question, let me know 😉